phpunit/functional/TicketTest.php

```diff
@@ -6953,9 +6954,42 @@ protected function timelineItemsProvider(): iterable
     {
         $now = time();
 
-        $postonly_user_id = getItemByTypeName(\User::class, 'post-only', true);
-        $normal_user_id   = getItemByTypeName(\User::class, 'normal', true);
-        $tech_user_id     = getItemByTypeName(\User::class, 'tech', true);
+        $postonly_user_id = getItemByTypeName(User::class, 'post-only', true);
+        $normal_user_id   = getItemByTypeName(User::class, 'normal', true);
+        $tech_user_id     = getItemByTypeName(User::class, 'tech', true);
+
+        $profile = getItemByTypeName(\Profile::class, 'Observer');
+
+        $profile_right = new \ProfileRight();
+        $profile_right->getFromDBByCrit([
+            'profiles_id' => $profile->getID(),
+            'name'        => 'task',
+        ]);
+
+        $this->updateItem(
```

Comment:
Do not change anything in database from a PHPUnit provider; it's persisted.

Reply:
It doesn't sound like a "real" provider, the way it's called:

![image](https://github.com/user-attachments/assets/a397a700-fd5a-4b6b-950e-c1c6b7176d29)

Reply:
Oh, indeed, my bad.